.github/workflows/delete-spam-issues.yml

```diff
@@ -0,0 +1,106 @@
+name: Delete Spam Issues
+
+on:
+  # Trigger when an issue is labeled as spam
+  issues:
+    types: [labeled]
+  # Also run manually from the Actions tab
+  workflow_dispatch:
+  # Run on a schedule to clean up any missed spam issues
+  schedule:
```

Comment:
how can it miss spam?

Reply:
I think if we end up with a lot of spam that the workflow ends up missing some. But I'm hoping we won't have to deal with that. This is just in case